draft/phoenix_presence_with_live_view.md

```diff
@@ -0,0 +1,146 @@
+# Using Phoenix Presence in LiveView |> A Simple Example
+
+Normally I'm not a big fan of things that seem like magic, but in the case of Phoenix Presence I actually love it. I love the way you can very simply add Phoenix Presence to your Phoenix app and you suddenly get tracking info on who's online at any given time. Amazing! I will definitely dig into the internals at some point in the future, but for now I wanted to show a very simple example of using Phoenix Presence inside LiveView!
```

Comment:
suggestion:

`I love the way you can very simply add Phoenix Presence to your Phoenix app and you suddenly get tracking info on who's online at any given time`

`I love the way you can very simply add Phoenix Presence to your Phoenix app and easily track user's presence with it`

or similar?